src/libsystemd/sd-bus/bus-convenience.c

```diff
@@ -44,9 +45,43 @@ _public_ int sd_bus_emit_signalv(
                         return r;
         }
 
+        if (destination) {
+                r = sd_bus_message_set_destination(m, destination);
```

Comment:
If we add the new function, then I think we should also introduce `sd_bus_message_new_signal_to()`, which also takes destination.

Reply:
Added it to the PR.